skyve-ext/src/main/java/org/skyve/EXT.java

```diff
@@ -574,6 +575,9 @@ else if ("pbkdf2".equals(passwordHashingAlgorithm)) {
 		else if ("scrypt".equals(passwordHashingAlgorithm)) {
 			result = "{scrypt}" + SCryptPasswordEncoder.defaultsForSpringSecurity_v5_8().encode(clearText);
 		}
+		else if ("argon2".equals(passwordHashingAlgorithm)) {
+			result = "{argon2}" + new Argon2PasswordEncoder(16, 32, 1, 19 * 1024, 2).encode(clearText);
```

Comment:
Can we use `Argon2PasswordEncoder.defaultsForSpringSecurity_v5_8()` instead? This way when Spring changes, we can use the new defaults. Otherwise it is unlikely we will increase these strength settings in Skyve.